src/main/java/io/thorntail/example/GreetingResource.java

```diff
@@ -13,30 +13,27 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package io.openshift.boosters;
+package io.thorntail.example;
 
-import java.net.InetAddress;
+import org.jboss.as.controller.client.ModelControllerClient;
```

Comment:
May be an import order issue ?

Reply:
I did reformat all imports, it's IntelliJ style now :-) `java.net.InetAddress` is no longer used.

Reply:
I guess it is not a problem `javax.` is at the bottom